docs/cluster-configuration.md

```diff
@@ -121,22 +118,29 @@ To enable a feature, add its name to the `featureGates` list and set it to `true
 disables the feature.
 
 ### hotplugVolumes Feature Gate
+
 Set the `hotplugVolumes` feature gate in order to allow attaching a data volume to a running VMI.
 
 ### withHostModelCPU Feature Gate
-Set the `withHostModelCPU`  feature gate in order to enable support migration for VMs with host-model CPU mode
+
+Set the `withHostModelCPU` feature gate in order to enable support migration for VMs with host-model CPU mode
 
 Additional information: [LibvirtXMLCPUModel](https://wiki.openstack.org/wiki/LibvirtXMLCPUModel)
 
 ### withHostPassthroughCPU Feature Gate
-Set the `withHostPassthroughCPU`  feature gate in order to allow migrating a virtual machine with CPU host-passthrough mode. 
+
+Set the `withHostPassthroughCPU` feature gate in order to allow migrating a virtual machine with CPU host-passthrough
+mode.
```

Comment:
Please explain why this is not simply true by default (you can add this in a follow-up PR, but then you should probably not touch these lines in this PR)

Reply:
@dankenigsberg we're in the middle of reviewing all the feature gares now, and we'll handle all of them in a separate PR.

Reply:
understood; that's why I don't think that whitespace noise helps here.

Reply:
+1